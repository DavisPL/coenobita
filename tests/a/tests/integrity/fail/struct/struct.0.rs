#![allow(unused_variables)]
#![allow(unused_assignments)]

struct BarBar<'a> {
    number: i32,
    
    #[cnbt::integrity({root}{root})]
    string: &'a str
}

fn main() {
    #[cnbt::integrity({root}{root,c})]
    let string = if c::boolean() {
        "something"
    } else {
        "else"
    };

    let b = BarBar {
        number: b::value(),
        string
    };
}