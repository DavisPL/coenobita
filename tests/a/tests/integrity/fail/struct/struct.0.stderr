error: expected {bin}{bin}, found {bin}{bin,c}
  --> tests/integrity/fail/struct/struct.0.rs:21:9
   |
21 |         string
   |         ^^^^^^
